src/main/java/org/shanerx/tradeshop/util/CustomItem.java

```diff
@@ -0,0 +1,142 @@
+ * to the respective contributor. No contributor should be held liable for
+ * any damages of any kind, whether be material or moral, which were
+ * caused by their contribution(s) to the project. See the full License for more information
+ */
+
+package org.shanerx.tradeshop.util;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.shanerx.tradeshop.TradeShop;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.logging.Level;
+
+public class CustomItem {
+
+    private static TradeShop plugin = (TradeShop) Bukkit.getPluginManager().getPlugin("TradeShop");
+    private static File file = new File(plugin.getDataFolder(), "customitems.yml");
+    private static FileConfiguration config = YamlConfiguration.loadConfiguration(file);
+    private static final char COLOUR_CHAR = '&';
+
+    public static void addItem(String name, ItemStack itm) {
+        if (!config.getValues(false).containsKey(name)) {
+            config.createSection(name);
+
+            config.set(name, itm.serialize());
+
+            save();
+        }
+    }
+
+    public static void removeItem(String name) {
+        if (config.getValues(false).containsKey(name)) {
+            config.set(name, null);
+
+            save();
+        }
+    }
+
+    public static Set<String> getItemSet() {
+        return config.getValues(false).keySet();
+    }
+
+    private static void setDefaults() {
+        if (config.getValues(false).isEmpty()) {
+            ItemStack dataHolder = new ItemStack(Material.TRIPWIRE_HOOK);
+            ItemMeta meta = dataHolder.getItemMeta();
+
+            meta.setDisplayName("Key");
+            meta.setLore(Arrays.asList("&aThe key to your dreams."));
+            dataHolder.setItemMeta(meta);
+
+            addItem("Key", dataHolder);
+        }
+    }
+
+    public static ItemStack getItem(String name) {
+        ItemStack itm = null;
+
+        if (!(config.get(name) == null)) {
+            itm = ItemStack.deserialize(config.getConfigurationSection(name).getValues(true));
+            ItemMeta meta = itm.getItemMeta();
+
+            if (meta.hasLore()) {
+                ArrayList<String> str2 = new ArrayList<>();
+                for (String s : meta.getLore()) {
+                    str2.add(ChatColor.translateAlternateColorCodes('&', s));
+                }
+                meta.setLore(str2);
+            }
+
+            if (meta.hasDisplayName())
+                meta.setDisplayName(ChatColor.translateAlternateColorCodes('&', meta.getDisplayName()));
+
+            itm.setItemMeta(meta);
+        }
+
+        return itm;
+    }
+
+    public static void reload() {
```

Comment:
Same here, add this as a non-static method to the CustomItemManager class... it would improve code quality and plugin architecture/algorithms.